evcxr_runtime/Cargo.toml

```diff
@@ -5,7 +5,10 @@ license = "Apache-2.0"
 description = "Runtime core for the evcxr crate"
 repository = "https://github.com/google/evcxr"
 readme = "README.md"
-authors = ["David Lattimore <dml@google.com>"]
+authors = [
+    "David Lattimore <dvdlttmr@google.com>",
```

Comment:
Looks like you accidentally changed part of my email address. This address will likely bounce, since it's not at gmail.com.

Reply:
Ah, I see you just copied my address from some other files, where I'd put my email address in incorrectly. I'll just merge this then fix my address everywhere.